slang.h

```diff
@@ -4425,6 +4441,8 @@ namespace slang
 
         virtual SLANG_NO_THROW void SLANG_MCALL setIgnoreCapabilityCheck(bool value) = 0;
 
+        virtual SLANG_NO_THROW SlangResult SLANG_MCALL getCompileTimeProfile(ISlangProfiler** compileTimeProfile, bool isClear) = 0;
```

Comment:
Maybe name it as "shouldClear", and add a comment on what it means -- "return a copy of internal profiling results, and if shouldClear is true, clear the profiling counters before returning."

Reply:
done